packages/web-react/src/components/Tabs/TabLink.tsx

```diff
@@ -1,21 +1,31 @@
 import React from 'react';
-import { ChildrenProps, TransferProps } from '../../types';
+import classNames from 'classnames';
+import { ChildrenProps, StyleProps } from '../../types';
+import { useStyleProps } from '../../hooks';
 import { useTabsStyleProps } from './useTabsStyleProps';
 
-export interface TabLinkProps extends ChildrenProps, TransferProps {
+export type TabLinkItemProps = StyleProps & React.HTMLProps<HTMLLIElement>;
+
+export interface TabLinkProps extends ChildrenProps {
   href: string;
+  itemProps?: TabLinkItemProps;
 }
 
-const TabLink = ({ children, href, ...restProps }: TabLinkProps): JSX.Element => {
+const TabLink = ({ children, href, itemProps = {}, ...restProps }: TabLinkProps): JSX.Element => {
   const { classProps } = useTabsStyleProps();
+  const { styleProps, props: itemTransferProps } = useStyleProps(itemProps);
 
   return (
-    <li className={classProps.item}>
+    <li {...styleProps} {...itemTransferProps} className={classNames(classProps.item, styleProps.className)}>
```

Comment:
DS-974 and DS-977 created. 

I would maybe prefer to add prefix to styleProps too. 

```suggestion
    <li {...itemStyleProps} {...itemTransferProps} className={classNames(classProps.item, itemStyleProps.className)}>
```

Reply:
`styleProps` changed to `itemStyleProps`